README.md

```diff
@@ -1,371 +1,55 @@
-# nginx-upload-module
+nginx-upload-module
+===================
 
-[![Build Status](https://travis-ci.org/fdintino/nginx-upload-module.svg?branch=master)](https://travis-ci.org/fdintino/nginx-upload-module)
-[![codecov](https://codecov.io/gh/fdintino/nginx-upload-module/branch/master/graph/badge.svg)](https://codecov.io/gh/fdintino/nginx-upload-module)
+This module is based on Nginx upload module (v 2.2.0) http://www.grid.net.ru/nginx/upload.en.html. ...
+Since it seems the author has not maintained that module. I changed some codes that can be installed with latest nginx.
```

Comment:
```suggestion
(hongzhidao) Since it seems the author has not maintained that module. I changed some codes that can be installed with latest nginx.
```